content/general/concepts/data-science/data-science.md

```diff
@@ -28,7 +28,7 @@ Since the beginning of the 21st century, data science has been used in generally
 - Data Engineering
 - Machine Learning & Deep Learning
 - Artificial Intelligence
-- Cloud and Distributed Computing
+- [Cloud and Distributed Computing](https://www.codecademy.com/resources/docs/general/cloud-computing)
 - Business Intelligence and Strategy
 
 ## Languages and Tools
```

Comment:
For the list below, I suggest the following:

- Make a nested list under "Python" and list out what's in the parentheses (Mathplotlib, pandas, NLTK, etc.).
  - Then link out to the Python, Mathplotlib, and pandas topics.
- Link out to R and SQL, as well.